apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -69,6 +68,47 @@ static SessionPool()
                     Task.Run(ReleaseAll).Wait(s_shutDownTimeout);
         }
 
+        // ReSharper disable once UnusedMember.Global
+        // returns true if the pool has evenly split its contents.
+        // For test purposes only.
+        // poolcontents will be filled with the contents of the pool and may not be null.
+        internal static bool GetPoolInfo(StringBuilder poolContents)
```

Comment:
Potential alternative return value: the difference between Max(size) and Min(size). (So the current return value is true if that would return 1 or 0, and false otherwise).